netjsonconfig/backends/base/backend.py

```diff
@@ -62,7 +62,7 @@ def _load(self, config):
                 pass
         if not isinstance(config, dict):
             raise TypeError(
-                'config block must be an istance ' 'of dict or a valid NetJSON string'
+                'config block must be an instance of dict or a valid Net JSON string'
```

Comment:
I see a new space between `Net` & `JSON`? `Net JSON` is one word, checkout: NetJSON: http://netjson.org/ , could you please revert it? :smile: